docs/moment/07-customization/17-invalid-date.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Invalid Date
+version: 2.3.0

```

Comment:
I can see this functionality was introduced in https://github.com/moment/moment/pull/1140 which was released in [2.3.0](https://github.com/moment/moment/releases/tag/2.3.0)